File: prak python/Vehicle.py
class Vehicle:
    def __init__(self, license_plate, brand, production_year, color):
        self._license_plate = license_plate
        self._brand = brand
        self._production_year = production_year
        self._color = color

    @property
    def license_plate(self):
        return self._license_plate

    @property
    def brand(self):
        return self._brand

    @property
    def production_year(self):
        return self._production_year

    @property
    def color(self):
        return self._color

    @license_plate.setter
    def license_plate(self, license_plate):
        self._license_plate = license_plate

    @brand.setter
    def brand(self, brand):
        self._brand = brand

    @production_year.setter
    def production_year(self, production_year):
        self._production_year = production_year

    @color.setter
    def color(self, color):
        self._color = color

File: prak cpp/Vehicle.cpp
#pragma once

#include <iostream>
#include <string>

using namespace std;

class Vehicle {
private:
    string _license_plate;
    string _brand;
    int _production_year;
    string _color;

public:
    Vehicle(string license_plate, string brand, int production_year, string color)
        : _license_plate(license_plate), _brand(brand), _production_year(production_year), _color(color) {}

    virtual ~Vehicle() {}

    string get_license_plate() const {
        return _license_plate;
    }

    string get_brand() const {
        return _brand;
    }

    int get_production_year() const {
        return _production_year;
    }

    string get_color() const {
        return _color;
    }

    void set_license_plate(string license_plate) {
        _license_plate = license_plate;
    }

    void set_brand(string brand) {
        _brand = brand;
    }

    void set_production_year(int production_year) {
        _production_year = production_year;
    }

    void set_color(string color) {
        _color = color;
    }
};